ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+The OCF maintains a strict policy of not removing or modifying content
+hosted by user accounts unless legally compelled to do so, or if such
+content is in violation of existing OCF account policies.
```

Comment:
I prefer the original wording because we shouldn't mess with user data even with "explicit permission" (see later comment)